lib/MusicBrainz/Server/Edit/Relationship/Delete.pm

```diff
@@ -117,6 +121,7 @@ sub build_display_data
     my $entity0_type = $link_type->{entity0_type};
     my $entity1_type = $link_type->{entity1_type};
     my $link = MusicBrainz::Server::Entity::Link->new(
+        $link_type->{id} ? (type_id => $link_type->{id}) : (),
```

Comment:
Not sure when this can be undefined, but it might break if so. Do we have a good way to find edits that might have this problem? I don't know if we can add conditions to the JSON in edit_data.data, but if we can I don't know how!